eZ/Publish/Core/Repository/Tests/Service/Mock/Base.php

```diff
@@ -68,7 +70,9 @@ protected function getRepository(array $serviceSettings = [])
                 $this->getRelationProcessorMock(),
                 $this->getFieldTypeRegistryMock(),
                 $this->createMock(PasswordHashServiceInterface::class),
-                $serviceSettings,
+                new FieldRegistry(),
```

Comment:
This is a clear indicator we should have an interface for FieldRegistry to be able to use mock here.